internal/mode/static/licensing/collector.go

```diff
@@ -49,30 +48,18 @@ func NewDeploymentContextCollector(
 
 // Collect collects all the information needed to create the deployment context for N+ licensing.
 func (c *DeploymentContextCollector) Collect(ctx context.Context) (dataplane.DeploymentContext, error) {
-	clusterInfo, err := telemetry.CollectClusterInformation(ctx, c.cfg.K8sClientReader)
-	if err != nil {
-		return dataplane.DeploymentContext{}, fmt.Errorf("error getting cluster information: %w", err)
+	depCtx := dataplane.DeploymentContext{
+		Integration:    integrationID,
+		InstallationID: c.cfg.PodUID,
 	}
 
-	var installationID string
-
-	// InstallationID is not required by the usage API, so if we can't get it, don't return an error
-	replicaSet, err := telemetry.GetPodReplicaSet(ctx, c.cfg.K8sClientReader, c.cfg.PodNSName)
+	clusterInfo, err := telemetry.CollectClusterInformation(ctx, c.cfg.K8sClientReader)
 	if err != nil {
-		c.cfg.Logger.Error(err, "failed to get NGF installationID")
-	} else {
-		installationID, err = telemetry.GetDeploymentID(replicaSet)
```

Comment:
If this function is no longer needed outside the telemetry package, let's unexport it.